lib/commons/matches/from-function.js

```diff
@@ -0,0 +1,26 @@
+/* global matches */
+
+/**
+ * Check if the value from a function matches some condition
+ * 
+ * Each key on the matcher object is passed to getValue, the returned value
+ * must than match the with the value of that matcher
```

Comment:
This text could be improved: `Each key on the matcher object is passed to getValue, the returned value
must match with the value of that matcher`

It would also benefit from an example like the other methods.